src/web/lib/components/AppHeader/index.scss

```diff
@@ -57,12 +57,21 @@
     padding: 0;
     width: 64px;
 
+    span {
```

Comment:
Undo all `pointer-events` changes here. I suppose that you did it to make sure that `event.target` pointed to the button instead of their descendant nodes, but if you add the click handler itself, then it is not needed.

As an alternative to making the child element ineligible for becoming a click target through CSS, you could also have used JS to detect whether the clicked element matches. So instead of `if (event.target.title === "Revert All")` you could have used `if (event.target.closest("[title='Revert All']")) {` (if the element is found, the matching element is returned, otherwise `null`).